e2e-tests/arbiter/run

```diff
@@ -36,12 +39,14 @@ check_cr_config() {
 		| grep '"host" :' \
 		| grep "$arbiter_ip"
 
-	# check read and write
+	desc 'Create user'
```

Comment:
I'm a bit confused. Do we want to use small or large letter first?

Reply:
nee to use small